Blizzard.IO.RabbitMQ/Entities/Rpc/RpcConfiguration.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Blizzard.IO.RabbitMQ.Entities.Rpc
```

Comment:
As you said, this project suppose to represent an implementation of `Blizzard.IO.Core`
its fine to create an `Entities` folder, but I don't see a special need for an `Rpc` folder, its suppose to be just like any other entity